capa/ida/plugin/icon.py

```diff
@@ -1,13 +1,7 @@
 )
-
-pixmap = QtGui.QPixmap()
-pixmap.loadFromData(ICON)
-QICON = QtGui.QIcon(pixmap)
```

Comment:
Source of the exception - TLDR; it appears code found in the global scope of a plugin and its dependencies is executed even though it does not appear IDA calls a plugin's `init` method when hosted under `idat`.

So at some point `capa_explorer.py` is imported which imports `capa/ida/plugin/form.py` which imports `capa/ida/plugin/icon.py` which executes the referenced code. This code is invalid under `idat` as the `Qt` libraries are not loaded.

Reply:
Can confirm by executing `traceback.print_stack()` when running IDA hosted under `idat`

![Screen Shot 2023-03-01 at 2 51 38 PM](https://user-images.githubusercontent.com/42192796/222273179-ab839db8-f782-47c9-b78c-105e529d47f5.png)

Reply:
Lesson learned: code stored at global scope of plugin and imports is executed even if the plugin returns `PLUGIN_SKIP` when its `init` method is called.